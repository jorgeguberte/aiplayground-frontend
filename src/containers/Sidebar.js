import React from 'react';
import ReactDOM from 'react-dom';

import '../styles/Sidebar.css';

export default class Sidebar extends React.Component{
    render(){
        return (
            <div className="Sidebar">
            </div>
        );
    }
}