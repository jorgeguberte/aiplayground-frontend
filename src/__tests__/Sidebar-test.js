import React from 'react';
import {shallow, mount, render} from 'enzyme';
import renderer from 'react-test-renderer';

import Sidebar from '../containers/Sidebar';

describe('integrity', ()=>{
    const component = renderer.create(<Sidebar/>);
    const mountedWrapper = mount(<Sidebar/>);

    test('Should render', ()=>{
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
})