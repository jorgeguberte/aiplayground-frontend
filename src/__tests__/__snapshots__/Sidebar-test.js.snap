// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`integrity Should render 1`] = `
<div
  className="Sidebar"
/>
`;
